packages/terms-of-service/src/class-terms-of-service.php

```diff
@@ -46,7 +46,7 @@ public function reject() {
 		 *
 		 * @since 7.9.0
 		 */
-		do_action( 'jetpack_reject_to_terms_of_service' );
+		do_action( 'jetpack_reject_terms_of_service' );
```

Comment:
This would change an action "in the wild" but, frankly, I'm okay doing so this quickly after release. Should update the `@since` to 7.9.x once we nail down the version number this will go out in.

Reply:
I think it's a safe bet that nobody is using this hook yet